low-level/files/opt/seagate/sspl/setup/setup_rabbitmq_cluster.py

```diff
@@ -188,6 +188,8 @@ def make_rmq_cluster(self):
                 print(f"RabbitMQ cluster is formed only with current host: {node_fqdn}")
 
     def process(self):
+        if not self.requested_nodes:
+            return
```

Comment:
You can also print a message saying that it is a single node vm.

Reply:
agree

Reply:
Okay.